src/libstore/derivations.cc

```diff
@@ -454,6 +475,8 @@ static const DrvHashModulo & pathDerivationModulo(Store & store, const StorePath
     return h->second;
 }
 
+UnknownHashes unknownHashes;
```

Comment:
Indeed we don't. I added it because I had forgotten that it should be `UnknownHashes {}` and not `UnknownHashes` if I wanted to inline it so it was failing with some obscure error :facepalm: 

Fixed in 3c723d3be